tests/run-make/raw-dylib-elf-verbatim-absolute/rmake.rs

```diff
@@ -0,0 +1,16 @@
+//@ only-elf
+//@ needs-dynamic-linking
+
+//! Ensure ELF raw-dylib is able to link against a non-existent verbatim absolute path
+//! by embedding the absolute path in the DT_SONAME and passing a different path for
+//! the linker for the stub.
+
+use run_make_support::{build_native_dynamic_lib, cwd, diff, rfs, run, rustc};
+
+fn main() {
+    // We compile the binary without having the library present.
+    // The verbatim library name is an absolute path.
+    rustc().crate_type("bin").input("main.rs").run();
```

Comment:
You'll need to additionally specify `.target(target())` (`run_make_support::target`) I think

Reply:
i did
> //@ ignore-cross-compile: Runs a binary.
which is much cooler imo